e2e/testcases/custom_resource_definitions_test.go

```diff
@@ -193,7 +214,8 @@ func mustRemoveUnManagedCustomResource(nt *nomostest.NT, dir string, crd string)
 	// TODO: Fix the multi-repo metrics error.
 	// Validate multi-repo metrics.
 	//err = nt.ValidateMetrics(nomostest.MetricsLatestCommit, func() error {
-	//	err := nt.ValidateMultiRepoMetrics(nomostest.DefaultRootReconcilerName, 3,
+	//	err := nt.ValidateMultiRepoMetrics(nomostest.DefaultRootReconcilerName,
+	//		nt.DefaultRootSyncObjects()+2, // 2 for the test Namespace & CustomResourceDefinition
 	//		metrics.ResourceCreated("CustomResourceDefinition"),
 	//		metrics.ResourceCreated("Namespace"))
 	//	return err
```

Comment:
Are we able to re-enable the validation with the fixes?

Reply:
maybe. i didn't try, just updated the expectation. since there's so much in this PR i figured it could wait.

Reply:
I'll try in a followup pr